docs/source/onnxruntime/usage_guides/amdgpu.mdx

```diff
@@ -42,12 +42,11 @@ pip install cmake onnx
 curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
 
 # Install ONNXRuntime from source
-git clone --recursive  https://github.com/ROCmSoftwarePlatform/onnxruntime.git
+git clone --single-branch --branch main --recursive https://github.com/Microsoft/onnxruntime onnxruntime
 cd onnxruntime
-git checkout rocm5.7_internal_testing_eigen-3.4.zip_hash
 
-./build.sh --config Release --build_wheel --update --build --parallel --cmake_extra_defines ONNXRUNTIME_VERSION=$(cat ./VERSION_NUMBER) --use_rocm --rocm_home=/opt/rocm
-pip install build/Linux/Release/dist/*
+./build.sh --config Release --build_wheel --allow_running_as_root --update --build --parallel --cmake_extra_defines CMAKE_HIP_ARCHITECTURES=gfx90a,gfx942 ONNXRUNTIME_VERSION=$(cat ./VERSION_NUMBER) --use_rocm --rocm_home=/opt/rocm
```

Comment:
`CMAKE_HIP_ARCHITECTURES=gfx90a,gfx942` is only for MI210/MI250/MI300 I think

Reply:
Added comment:

> Note: The instructions build ORT for `MI210/MI250/MI300` gpus. To support other architectures, please update the `CMAKE_HIP_ARCHITECTURES` in the build command.